staging/src/k8s.io/apiserver/pkg/storage/testing/watcher_tests.go

```diff
@@ -1277,6 +1305,14 @@ func RunWatchSemantics(ctx context.Context, t *testing.T, store storage.Interfac
 				return []watch.Event{initialEventsEndFromLastCreatedPod(createdInitialPods)}
 			},
 		},
+		{
+			name:                               "allowWatchBookmarks=true, sendInitialEvents=false, RV=1",
+			allowWatchBookmarks:                true,
+			sendInitialEvents:                  &falseVal,
+			resourceVersion:                    "1",
+			initialPods:                        []*example.Pod{makePod("1"), makePod("2")},
+			expectedInitialEventsInRandomOrder: addEventsFromCreatedPods,
```

Comment:
Why?

For sendInitialEvents=false, we don't expect initial events ?

What we rather expect is the actual stream of events (so should be strict order).

Reply:
no, I don't think so, we expect events > `1` (this is different than initial events) which are taken from the map so the order is not guaranteed.

Reply:
We're not taking from the map - we're only taking the state from the map.
Here we're actually taking a watch stream.

Reply:
watch stream = transaction log

Reply:
ah, now i see you what you mean, we actually have "two maps", I will double check that

Reply:
okay, you were right, thanks!